.env.example

```
@@ -143,6 +143,7 @@ CONNECTION_LLM_REQUIRED=0.3                # Similarity >N = needs LLM (35% PAID
 # Used in priority order: 1) Agent-specific custom key, 2) Global settings key, 3) Environment variable
 
 ANTHROPIC_API_KEY=              # Anthropic API key (starts with sk-ant-)
+ATLASCLOUD_API_KEY=             # Atlas Cloud API key
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)